unity/unity_utils.c

```diff
@@ -278,24 +278,30 @@ mono_unity_class_get_generic_parameter_count (MonoClass* klass)
 	return klass->generic_container->type_argc;
 }
 
-static char* data_dir = NULL;
-void
-mono_unity_set_data_dir(const char* dir)
-{
-	if (data_dir)
-		g_free(data_dir);
-
-	data_dir = g_new(char*, strlen(dir) + 1);
-	strcpy(data_dir, dir);
-}
- 
-char*
-mono_unity_get_data_dir()
-{
-	return data_dir;
-}
-
-MonoClass* mono_unity_class_get(MonoImage* image, guint32 type_token)
-{
-	return mono_class_get(image, type_token);
-}
+static char* data_dir = NULL;
+void
+mono_unity_set_data_dir(const char* dir)
```

Comment:
What's up with the whitespace changes?

Reply:
Yeah I was wondering about that myself and tried to revert it, but vscode "fixed" it on save again, so I suspect it was lineendings, not sure how that's even possible with git :confused: